### PR TITLE
Tests for iswritable for TCPSocket

### PR DESCRIPTION
**Pull request by kshyatt**
Welcome to 2019! The [coverage build](https://codecov.io/gh/kshyatt/julia/src/master/stdlib/Sockets/src/Sockets.jl) I ran suggested these lines weren't covered. The test passed locally for me, hopefully it is ok.

### CODE REVIEW

File: stdlib/Sockets/test/runtests.jl [hunk 1]
```diff
@@ -413,6 +413,18 @@ end
...
 end
 
+@testset "iswritable" begin
+    let addr = Sockets.InetAddr(ip"127.0.0.1", 4445)
+        srv = listen(addr)
+        s = Sockets.TCPSocket()
+        Sockets.connect!(s, addr)
+        @test iswritable(s)
+        r = @async close(s)
+        fetch(r)
```
Comment (vtjnash) [inline stdlib/Sockets/test/runtests.jl]:
```suggestion
        close(s)
```

File: stdlib/Sockets/test/runtests.jl [hunk 2]
```diff
@@ -413,6 +413,18 @@ end
     end
 end
 
+@testset "iswritable" begin
+    let addr = Sockets.InetAddr(ip"127.0.0.1", 4445)
+        srv = listen(addr)
+        s = Sockets.TCPSocket()
+        Sockets.connect!(s, addr)
+        @test iswritable(s)
+        r = @async close(s)
```
Comment (vtjnash) [inline stdlib/Sockets/test/runtests.jl]:
```suggestion
```

**Review by vtjnash — APPROVED** [review]

**Comment by vtjnash** [comment]
Does `isreadable` get covered? Seems like it would be appropriate to test that function here also

**Comment by kshyatt** [comment]
`isreadable` does get covered incidentally in other tests

**Comment by kshyatt** [comment]
Did I screw up AV or is that something else?

**Comment by kshyatt** [comment]
OK looks like it was another failure like #30551...

**Pull request merged**